demos/shape.html

```diff
@@ -46,7 +46,7 @@
           <div class="mdc-shape-container__corner mdc-shape-container__corner--bottom-left"></div>
         </div>
         <div class="mdc-shape-container four-corner-container four-corner-container--stroked">
-          <button class="mdc-button mdc-button--stroked">Stroked Button</button>
+          <button class="mdc-button mdc-button--outlined">Stroked Button</button>
```

Comment:
The button caption here still says Stroked

Reply:
Yes, this was updated in the shape PR.